bundle/manifests/rhdh.redhat.com_orchestrators.yaml

```diff
@@ -546,6 +569,16 @@ spec:
                         type: string
                     type: object
                 type: object
+              sonataflow:
```

Comment:
This section does not exist in values.yaml file and should not be here.

Reply:
@masayag is it OK to remove this section which is used only for RC? It's not being used in the `values.yaml`.

Reply:
I don't mind removing it, as long as there is an option to configure the orchestrator to use RC builds of OSL (and RHDH).

Reply:
Agreed to remove the RC references in main.

Reply:
So at the end I will keep the schema spec but remove the RC related value files since removing it from the schema at this point would break compatibility with v1alpha1. I will introduce these changes in the v1alpha2.
@jenniferubah FYI. Once your PR is merged I will raise a new one to cleanup the `RC` artifacts in the schema.